hudi-hadoop-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -459,7 +457,7 @@ public void testHugeLogFileWrite() throws IOException, URISyntaxException, Inter
     header.put(HoodieLogBlock.HeaderMetadataType.SCHEMA, getSimpleSchema().toString());
     byte[] dataBlockContentBytes = getDataBlock(DEFAULT_DATA_BLOCK_TYPE, records, header).getContentBytes();
     HoodieLogBlock.HoodieLogBlockContentLocation logBlockContentLoc = new HoodieLogBlock.HoodieLogBlockContentLocation(
-        HadoopFSUtils.getStorageConf(new Configuration()), null, 0, dataBlockContentBytes.length, 0);
+        HoodieStorageUtils.getNewStorageConf(), null, 0, dataBlockContentBytes.length, 0);
```

Comment:
use `getDefaultStorageConf`.

Reply:
getDefaultStorageConf does
```
new Configuration(false)
```

Reply:
put this into a test util method because we don't want prod code to use empty config